sdk/core/core-http/lib/util/xml.browser.ts

```diff
@@ -1,16 +1,32 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+import { removeBOM } from "./utils";
+
 // tslint:disable-next-line:no-null-keyword
 const doc = document.implementation.createDocument(null, null, null);
 
 const parser = new DOMParser();
-export function parseXML(str: string): Promise<any> {
+export function parseXML(str: string, opts?: { includeRoot?: boolean }): Promise<any> {
   try {
+    if (str == undefined || str == "") {
+      throw new Error("Document is empty");
+    }
+
+    if (!removeBOM(str).startsWith("<")) {
+      throw new Error("Non-whitespace before first tag");
```

Comment:
Was this exception been thrown before? If not, this might break existing customers when they use this code.